opencti-platform/opencti-graphql/tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "noImplicitAny": true,
     "noFallthroughCasesInSwitch": true,
     "moduleResolution": "node",
+    "resolveJsonModule": true,
```

Comment:
why? should we keep this?

Reply:
it's to import .json files (the data file for the tests) in TS.
So far, all test files where in JS.